### PR TITLE
Add prefix support for AWS Queues

### LINKED ISSUE #521: Allow specify queues name
https://github.com/danielgerlag/workflow-core/blob/5bf34b1a69d22056f5563e5884165f8a3106be32/src/providers/WorkflowCore.Providers.AWS/Services/SQSQueueProvider.cs#L57

Hello, please. Allow pass queue prefix/name as parameters.

Thanks.

### PR DESCRIPTION
**Pull request by keelerm84**
Closes #521

### CODE REVIEW

File: src/providers/WorkflowCore.Providers.AWS/ServiceCollectionExtensions.cs [hunk 1]
```diff
@@ -13,9 +13,9 @@ namespace Microsoft.Extensions.DependencyInjection
 {
     public static class ServiceCollectionExtensions
     {
-        public static WorkflowOptions UseAwsSimpleQueueService(this WorkflowOptions options, AWSCredentials credentials, AmazonSQSConfig config)
+        public static WorkflowOptions UseAwsSimpleQueueService(this WorkflowOptions options, AWSCredentials credentials, AmazonSQSConfig config, string queuesPrefix = "workflowcore")
```
Comment (keelerm84) [inline src/providers/WorkflowCore.Providers.AWS/ServiceCollectionExtensions.cs]: I set a default here to ensure backwards compatibility. Let me know if you would prefer I remove that.

File: src/providers/WorkflowCore.Providers.AWS/Services/SQSQueueProvider.cs [hunk 2]
```diff
@@ -54,9 +56,9 @@ public async Task<string> DequeueWork(QueueType queue, CancellationToken cancell
 
         public async Task Start()
         {
-            var workflowQueue = await _client.CreateQueueAsync(new CreateQueueRequest("workflowcore-workflows"));
-            var eventQueue = await _client.CreateQueueAsync(new CreateQueueRequest("workflowcore-events"));
-            var indexQueue = await _client.CreateQueueAsync(new CreateQueueRequest("workflowcore-index"));
+            var workflowQueue = await _client.CreateQueueAsync(new CreateQueueRequest($"{_queuesPrefix}-workflows"));
```
Comment (keelerm84) [inline src/providers/WorkflowCore.Providers.AWS/Services/SQSQueueProvider.cs]: Would you prefer the hyphen be included in the default value as opposed to being hardcoded here?

**Comment by danielgerlag** [comment]
Thanks, I'm currently traveling but I will try get access to a computer asap.

**Comment by danielgerlag** [comment]
@keelerm84  Sorry for the delay, I was stranded in a foreign country for 3 months... Thanks for your contribution... could you also please increment the version numbers of all the libraries that need to be deploy to nuget?

**Comment by keelerm84** [comment]
> I was stranded in a foreign country for 3 month

I saw the ticket you had filed. That must have been incredibly difficult.

> could you also please increment the version numbers of all the libraries that need to be deploy to nuget?

I'm still pretty new to C# and I haven't had to deal with package deployments before. AFAIK I should just have to check the version numbers in the `src/providers/WorkflowCore.Providers.AWS/WorkflowCore.Providers.AWS.csproj` file, right?

Since I added a new parameter to the SQSProvider class, which isn't internal only, I need to bump the version to 4.0.0?

**Comment by danielgerlag** [comment]
I would make it 3.0.2

**Pull request merged**